tools/Test/SmokeTest/ScriptsCaller.ps1

```diff
@@ -8,6 +8,20 @@ param(
   $script
 )
 
+$IsLinuxEnv = (Get-Variable -Name "IsLinux" -ErrorAction Ignore) -and $IsLinux
+$IsMacOSEnv = (Get-Variable -Name "IsMacOS" -ErrorAction Ignore) -and $IsMacOS
+$IsWinEnv = !$IsLinuxEnv -and !$IsMacOSEnv
+
+if (-not $DestinationPowerShell) {
+  if ($IsWinEnv) {
+    $DestinationPowerShell = "D:\a\_work\1\s"
```

Comment:
Move them to common.ps1 please so that we don't need to maintain constants in ScriptsCaller.ps1 and PrepareRequiredPowerShell.ps1 at the same time.